docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,52 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+Debugging Interface will help user to debug the Bear code and move through
+it using ``pdb`` Interface
+
+There are two way to invoke the coala's Debgger
+
+Command Line Interface
+----------------------
+
+User can specify the bear they want to debug using ``--debug-bears`` i.e.,
+
+.. code:: shell
+
+    $ coala -b PEP8Bear,MypyBear -f <filename> --debug-bears PEP8Bear
+
+If bear name is not specified in ``--debug-bears`` argument then it will by
+default debug all the bears passed through ``--bears`` or ``-b`` argument
+**i.e.,**
+
+.. code:: shell
+
+    $ coala -b PEP8Bear,MypyBear -f <filename> --debug-bears
+
+Configuration Files - coafiles
+------------------------------
+
+User can debug the bears using ``.coafile`` **i.e.,**
+
+::
+
+    [<section name>]
+    bears = PEP8Bear,MypyBear
+    files = <filename>
+    debug_bears = PEP8Bear
+
+**or**
+
+::
+
+    [<section name>]
+    bears = PEP8Bear,MypyBear
+    files = <filename>
+    debug_bears = True
+
+.. note::
+
+    ``Pdb`` ``do_q`` (quit) has been remapped so that coala can continue its
```

Comment:
Nobody knows `do_q`, that's implementation detail.

Reply:
So -->

Pdb's quit-command (`q`) has been remapped to coala continues its normal execution without aborting.

(note also: in this case arguable, but I wouldn't wrap Pdb into backticks, since Pdb is also referred to as a name for Python's debugging capabilities. This is indeed a blurry border, but imo it fits better without backticks.)